swan-lake/development-tutorials/deployment-guide/serverless-deployment.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: ballerina-cloud-left-nav-pages-swanlake
+title: Serverless deployment
+description: Serverless architecture allows developers to focus on writing application logic without worrying about managing servers or infrastructure. AWS Lambda automatically handles provisioning, scaling, and maintaining resources, letting you concentrate solely on your code. Following are the  Ballerina functions in a serverless environment, 
+keywords: ballerina, programming language, services, cloud, kubernetes, docker
+active: ballerina-deployment-guideLines
+intro: Serverless architecture allows developers to focus on writing application logic without worrying about managing servers or infrastructure. AWS Lambda automatically handles provisioning, scaling, and maintaining resources, letting you concentrate solely on your code. Following are the  Ballerina functions in a serverless environment, 
+---
+
+1. [AWS Lambda](#aws-lambda)
+2. [Azure Functions](#azure-function)
+
+## AWS Lambda
+
+In this guide, we'll walk through deploying a serverless application using AWS Lambda with Ballerina. You'll learn how to set up, deploy, and manage your application, simplifying your production workflow and eliminating the need to manage servers.
+An AWS Lambda function can be triggered by various AWS services. You can find the list of supported notification types below.
+- Direct invocation
+- Simple Queue Service (SQS)
+- Simple Storage Service (S3)
+- DynamoDB 
+- Simple Email Service (SES)
+- API Gateway
```

Comment:
Add the links similar to https://ballerina.io/learn/aws-lambda/#supported-triggers

Reply:
Btw do we need the above list here as it is repeated in https://ballerina.io/learn/aws-lambda/ as well?